misc/docerina/src/main/resources/docerina-templates/html/index.hbs

```diff
@@ -1,74 +1,122 @@
-
-    <!-- Roboto font CSS -->
-    <link rel="stylesheet" href="docerina-theme/css/roboto.css">
-
-    <!-- Nanoscroller CSS -->
-    <link rel="stylesheet" href="docerina-theme/css/nanoscroller.css">
-
-    <!-- custom CSS -->
-    <link rel="stylesheet" href="docerina-theme/css/main.css">
-
-    <!-- Jquery JS -->
-    <script src="docerina-theme/js/jquery.min.js"></script>
-
-    <!-- Nanoscroller JS -->
-    <script src="docerina-theme/js/jquery.nanoscroller.min.js"></script>
-
-    <script type="text/javascript">
-        $(document).ready(function(){
-            $('.nano').nanoScroller();
-        });
-    </script>
-
-    <title>Ballerina API Documentation</title>
+   <meta charset="utf-8">
+   <meta http-equiv="X-UA-Compatible" content="IE=edge">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <meta name="author" content="WSO2, Inc.">
+   <link rel="shortcut icon" href="/img/favicon.ico">
+   <title>Ballerina.io - API Docs</title>
+   <script src="//code.jquery.com/jquery-3.3.1.min.js"></script>
+   <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+   <!-- Optional theme -->
+   <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
+   <!-- Latest compiled and minified JavaScript -->
+   <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+   <!--      <script src="../../js/jquery-2.1.1.min.js"></script>-->
+   <!--  <script src="../../js/modernizr-2.8.3.min.js"></script>
+      <script type="text/javascript" src="../../js/highlight.pack.js"></script>-->
+   <script src="/js/jquery-2.1.1.min.js"></script>
+   <script src="/js/modernizr-2.8.3.min.js"></script>
+   <script type="text/javascript" src="../../js/highlight.pack.js"></script>
+   <link rel="stylesheet" href="https://use.typekit.net/son4ymv.css"/>
+   <script src="/js/ballerina-common.js"></script>
+   <link rel="stylesheet" href="/css/theme.css" type="text/css" />
+   <link rel="stylesheet" href="/css/theme_extra.css" type="text/css" />
+   <link rel="stylesheet" href="/css/highlight.css">
+   <link rel="stylesheet" href="/css/ballerina-io.css">
+   <link rel="stylesheet" href="/css/ballerina-io-docs.css">
+   <link rel="stylesheet" href="/css/bbe-page.css">
+   <script src="docerina-theme/js/jquery.nanoscroller.min.js"></script>
+   <script type="text/javascript">
+      $(document).ready(function(){
+          $('.nano').nanoScroller();
+      });
+   </script>
+   <script>
+      // Current page data
+      var mkdocs_page_name = "ballerina-by-Examples";
```

Comment:
this name ends with capital ... is that fine? "ballerina-by-Examples"

Reply:
its not an URI :)